### ISSUE TITLE
Duplicate id's

### ISSUE THREAD

**Issue by MartijnLeplae** [fpsluozi/tofindwaldo#3]
There appear to be some duplicate id's in the splits.

I used following Python script to identify them:

`seen = set()`
`dupes = [x for x in ids if x in seen or seen.add(x)]`

Where ids is a list of all the ids in train, test and val combined.

Following file contains the duplicates I found: [dupes.txt](https://github.com/fpsluozi/tofindwaldo/files/11158410/dupes.txt)

Please correct me if I'm wrong on this.

**Comment by fpsluozi** [comment]
My bad. The duplicate ids are apparently from the val set. The training set and the test set are still intact.

val.txt has been updated accordingly. Thanks for pointing this out.
